SPECS/python27-alt.spec

```diff
@@ -25,9 +25,9 @@ Group: Development/Languages
 License: Python
 URL: http://www.python.org/
 Source: https://www.python.org/ftp/python/%{version}/Python-%{version}.tar.xz
-Obsoletes: python27 <= 2.7.6
-Provides: python27 = 2.7.6
-Provides: python(abi) = 2.7.6
+Obsoletes: python27 <= 2.7.11
+Provides: python27 = 2.7.11
+Provides: python(abi) = 2.7.11
```

Comment:
Above this, you should change the "Release:" line back to 1 instead of 2.  This is Release 1 of the 2.7.11 RPM.